src/tainting/Dataflow_tainting.ml

```diff
@@ -962,43 +968,71 @@ let rec check_tainted_lval env (lval : IL.lval) : Taints.t * Lval_env.t =
   report_findings { env with lval_env } findings;
   (taints, lval_env)
 
-and propagate_taint_via_unresolved_java_getters_and_setters env e args
+and propagate_taint_via_java_getters_and_setters_without_definition env e args
```

Comment:
it's not very clear to me what the objective of this function is, could there be a comment?